pkg/util/cluster/cluster.go

```diff
@@ -135,14 +135,32 @@ func (c *Cluster) Create(ctx context.Context, vnetResourceGroup, clusterName str
 	}
 
 	c.log.Infof("creating AAD application")
-	appID, appSecret, err := c.createApplication(ctx, "aro-"+clusterName)
```

Comment:
Changes to this file were specific to a local development scenario and were accidentally committed. All changes to this file have been reverted.